CodeEdit/Features/Search/Views/QuickSearchResultLabel.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  QuickSearchResultLabel.swift
+//  CodeEdit
+//
+//  Copyright © 2024-present Paul Ebose. All rights reserved.
```

Comment:
Due to the license of the project, we cannot include copyrighted code. Can we ask you resubmit a new implementation or a version without the notice? It needs to replace this commit entirely otherwise it will still be in the commit history.

Reply:
I just noticed this and was coming to change it. How do you suggest I do this? a force commit?

Reply:
On that note, it will be nice if this project has a header comment template for new files.

Reply:
Something like this in `CodeEdit.xcodeproj/xcshareddata/IDETemplateMacros.plist`:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
<key>FILEHEADER</key>
<string>
//  ___FILENAME___
//  ___TARGETNAME___
//
//  Created by ___FULLUSERNAME___ on ___DATE___.
//</string>
</dict>
</plist>
```